packages/@ember/-internals/runtime/index.ts

```diff
@@ -2,20 +2,12 @@ export { default as RegistryProxyMixin } from './lib/mixins/registry_proxy';
 export { default as ContainerProxyMixin } from './lib/mixins/container_proxy';
 export { default as compare } from './lib/compare';
 export { default as isEqual } from './lib/is-equal';
-export {
-  default as Array,
-  NativeArray,
-  A,
-  MutableArray,
-  removeAt,
-  uniqBy,
-  isArray,
-} from './lib/mixins/array';
 export { default as Comparable } from './lib/mixins/comparable';
+export { NativeArray } from '@ember/array';
```

Comment:
We re-export here since there are tests that want this to exist in `@ember/-internals`. These should be revisited in the future.